pandas/tests/indexes/datetimes/test_arithmetic.py

```diff
@@ -355,6 +355,49 @@ def test_dti_isub_int(self, tz, one):
         rng -= one
         tm.assert_index_equal(rng, expected)
 
+    # -------------------------------------------------------------
+    # __add__/__sub__ with integer arrays
+
+    @pytest.mark.parametrize('freq', ['H', 'D'])
+    @pytest.mark.parametrize('box', [np.array, pd.Index])
+    def test_dti_add_intarray_tick(self, box, freq):
```

Comment:
we must have *some* tests that do index arithmetic with integers, either move them here if they are not redundant or remove them.